### ISSUE TITLE
Select all checkbox in Devices view doesn't select anything

### ISSUE THREAD

**Issue by tombe75** [netbox-community/netbox#7098]
### NetBox version

v3.0.0

### Python version

3.8

### Steps to Reproduce

1. Go to https://demo.netbox.dev/dcim/devices/
2. Click checkbox to mark all devices next to Name header
 

### Expected Behavior

All devices selected

### Observed Behavior

No devices get selected, also verified in private Netbox test instance.

**Comment by jeremystretch** [comment]
This looks like an exception in the Javascript stemming from an error where there exists a custom field with a space in the name. Deleting the custom field fixed the issue, but after recreating the custom field, I haven't been able to reproduce the error.

Someone on Slack also came across this. The error they observed in the browser console is below (where `support provider` is the name of the custom field).

```
Uncaught DOMException: Document.querySelector: 'label[for=id_cf_support provider]' is not a valid selector netbox.js:17
```

Again, I have been unable to reproduce this by creating a custom field either locally or on the public demo instance. I'm going to close out this issue, because it doesn't appear to have anything to do with the devices list specifically, but if someone can capture the steps needed to reproduce the issue, please open a new bug report with that information and we'll work on fixing it.

**Comment by jeremystretch** [comment]
Disregard the above; it seems to have been fixed in #7075 already.
